// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/compiler/modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -21,9 +21,8 @@
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(16,10): error TS9007: Function must have an explicit return type annotation with --isolatedDeclarations.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(17,5): error TS2339: Property 'name' does not exist on type '() => void'.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(20,6): error TS2550: Property 'sign' does not exist on type 'Math'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2015' or later.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(25,5): error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations.
-modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(25,5): error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(25,6): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(29,18): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(33,9): error TS9010: Variable must have an explicit type annotation with --isolatedDeclarations.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(33,13): error TS2304: Cannot find name 'Proxy'.
@@ -33,13 +32,12 @@
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(44,5): error TS2550: Property 'includes' does not exist on type 'string'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2015' or later.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(47,9): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(47,9): error TS9010: Variable must have an explicit type annotation with --isolatedDeclarations.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(51,5): error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations.
-modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(51,5): error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts(51,6): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
 
 
-==== modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts (22 errors) ====
+==== modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts (20 errors) ====
     // All will be error from using ES6 features but only include ES5 library
     // Using Es6 array
     function f(x: number, y: number, z: number) {
              ~
@@ -84,11 +82,8 @@
         ~~~~~~~~~~~~~~~~~~~~
 !!! error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations.
 !!! related TS9027 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts:23:5: Add a type annotation to the variable o.
 !!! related TS9034 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts:25:5: Add a return type to the method
-        ~~~~~~~~~~~~~~~~~~~~
-!!! error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
-!!! related TS9027 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts:23:5: Add a type annotation to the variable o.
          ~~~~~~
 !!! error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
             return false;
         }
@@ -140,11 +135,8 @@
         ~~~~~~~~~~~~~~~~~~~~
 !!! error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations.
 !!! related TS9027 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts:50:7: Add a type annotation to the variable o1.
 !!! related TS9034 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts:51:5: Add a return type to the method
-        ~~~~~~~~~~~~~~~~~~~~
-!!! error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
-!!! related TS9027 modularizeLibrary_ErrorFromUsingES6FeaturesWithOnlyES5Lib.ts:50:7: Add a type annotation to the variable o1.
          ~~~~~~
 !!! error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
             return false;
         }
